editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -486,6 +486,7 @@ export const GridControls = controlForStrategyMemoized<GridControlsProps>(({ tar
       store.editor.canvas.interactionSession != null &&
       store.editor.canvas.interactionSession.activeControl.type === 'GRID_CELL_HANDLE' &&
       store.editor.canvas.interactionSession?.interactionData.type === 'DRAG' &&
+      !store.editor.canvas.interactionSession?.interactionData.modifiers.cmd &&
```

Comment:
`store.editor.canvas.interactionSession?.interactionData.modifiers.cmd` can be undefined, so adding a `!` operator on that relies on truthiness, which we try to avoid. I would just write a `!== true` instead of this